packages/next/lib/load-custom-routes.ts

```diff
@@ -108,6 +108,8 @@ function checkHeader(route: Header): string[] {
 
   if (!Array.isArray(route.headers)) {
     invalidParts.push('`headers` field must be an array')
+  } else if (route.headers.length === 0) {
+    invalidParts.push('`headers` field can not be empty')
```

Comment:
```suggestion
    invalidParts.push('`headers` field cannot be empty')
```

https://www.merriam-webster.com/words-at-play/cannot-vs-can-not-is-there-a-difference